src/ResourceManager/Sql/Commands.Sql/Microsoft.Azure.Commands.Sql.format.ps1xml

```diff
@@ -474,5 +474,1134 @@
+            <Label>Description</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>StartTime</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>EndTime</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>ScheduleType</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>Interval</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>Enabled</Label>
+          </TableColumnHeader>
+        </TableHeaders>
+        <TableRowEntries>
+          <TableRowEntry>
+            <Wrap/>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>JobName</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>Version</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>Description</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>StartTime</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>EndTime</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>ScheduleType</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>Interval</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <PropertyName>Enabled</PropertyName>
+              </TableColumnItem>
+            </TableColumnItems>
+          </TableRowEntry>
+        </TableRowEntries>
+      </TableControl>
+    </View>
+    <View>
+      <Name>Microsoft.Azure.Commands.Sql.ElasticJobs.Model.AzureSqlElasticJobTargetGroupModel</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.Sql.ElasticJobs.Model.AzureSqlElasticJobTargetGroupModel</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableHeaders>
+          <TableColumnHeader>
+            <Label>TargetGroupName</Label>
+          </TableColumnHeader>
+          <TableColumnHeader>
+            <Label>Targets</Label>
+          </TableColumnHeader>
+        </TableHeaders>
+        <TableRowEntries>
+          <TableRowEntry>
+            <Wrap/>
+            <TableColumnItems>
+              <TableColumnItem>
+                <PropertyName>TargetGroupName</PropertyName>
+              </TableColumnItem>
+              <TableColumnItem>
+                <ScriptBlock>
+                  $_.Targets | % { "(" + $(If($_.MembershipType -eq "Exclude") { "-" }) + $_.TargetServerName + $(If($_.TargetElasticPoolName) { "." + $_.TargetElasticPoolName }) + $(If($_.TargetShardMapName) { "." + $_.TargetShardMapName }) + $(If($_.TargetDatabaseName) { "." + $_.TargetDatabaseName }) + ")"}
```

Comment:
ShardMapName & DatabaseName should be other way around. One shard management database can contain many shard maps.

Reply:
gotcha. will reverse.